types/openui5/sap.tnt.d.ts

```diff
@@ -50,20 +50,19 @@ declare module "sap/tnt/InfoLabel" {
    *
    * Overview:
    *
-   * The control visualizes text information without user interaction. The text inside the control is always
-   * in upper case. It can have smaller or larger side paddings which can be specified by the `renderMode`
-   * property. The text-background color pair can be changed by setting a number between 1 and 10 that corresponds
-   * to the 10 predefined color combinations of the `colorScheme` property. The control is designed to be
-   * vertically aligned with UI5 Input and Button control families. When using `InfoLabel` in non-editable
-   * `Forms`, `Tables`, etc., set `displayOnly=true` for best visual results.
+   * The control visualizes text information without user interaction. The text-background color pair can
+   * be changed by setting a number between 1 and 10 that corresponds to the 10 predefined color combinations
+   * of the `colorScheme` property. The control is designed to be vertically aligned with UI5 Input and Button
+   * control families. When using `InfoLabel` in non-editable `Forms`, `Tables`, etc., set `displayOnly=true`
+   * for best visual results.
    *
    * Usage Guidelines:
    * 	 - If the text is longer than the width of the control, it doesn’t wrap. Instead, it’s represented as
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/openui5/sap.tnt.d.ts;
  Line: 60;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.